src/presenter/drawing-markup-elements-presenter.js

```diff
@@ -0,0 +1,36 @@
+import UserTitle from '../view/user-title-view.js';
+import FilterElements from '../view/filter-view.js';
+import CardMovie from '../view/movie-card-view.js';
+import ShowMoreButton from '../view/button-show-more.js';
+import FilmContainer from '../view/film-container-view.js';
+import PopupForInformation from '../view/popup-view.js';
+import {render} from '../render.js';
+
+export default class DrawingMarkupElementsPresenter {
+
+  init = (renderingContainer) => {
+    this.containerForRendering = renderingContainer;
+
+    const headerLogo = document.querySelector('.header');
+    this.placeForTitleUser = headerLogo;
```

Comment:
я не понимаю, зачем создавать эти placeFor.... 
можно же использовать напрямую? или хотя бы так. Зачем создавать лищние переменные?
```suggestion
    this.placeForTitleUser = document.querySelector('.header');
```